lib/vanagon/platform/defaults/osx-10.15-x86_64.rb

```diff
@@ -0,0 +1,22 @@
+platform "osx-10.15-x86_64" do |plat|
+  plat.servicetype 'launchd'
+  plat.servicedir '/Library/LaunchDaemons'
+  plat.codename "catalina"
+
+  plat.provision_with 'export HOMEBREW_NO_EMOJI=true'
+  plat.provision_with 'export HOMEBREW_VERBOSE=true'
```

Comment:
Quoting-style " vs. ' mismatch. 

I think throughout all the platform-defaults files we should adopt a consistent quoting style. I'm on the fence on which would be better:

-  The typical Ruby: use single-quotes except when you must use double-quotes for variable expansion.

OR:

- (like most of the files here already): always use double quotes.

The first follows good Ruby form but looks a little awkward/fussy inside these DSL files. The second is easy and may end up looking more readable.

Reply:
I'll double quote everything for consistency/readability